GitDiffMargin/Git/GitCommands.cs

```diff
@@ -207,7 +213,7 @@ public void StartExternalDiff(ITextDocument textDocument)
                     leftLabel = string.Format("{0}@{1}", objectName, repo.Head.Tip.Sha.Substring(0, 7));
                 }
 
-                string rightLabel = filename;
+                string rightLabel = originalPath;

```

Comment:
:memo: This one is clever. It makes the diff window *look* like the original file is being edited, even though the buffer still maps to the temporary location.